### ISSUE TITLE
Export disappears from the screen before it can be downloaded

### ISSUE THREAD

**Issue by Lhorus6** [LuatixHQ/opencti#6759]
## Description

When we try to export entity (at least with the configuration describe in the repro case bellow), the export disappears directly when the export is done (before we can download it).

## Environment

OCTI 6.0.10

## Reproducible Steps

Steps to create the smallest reproducible scenario:
1. Go to Observables
2. Select 4 IPv4
3. Export them in JSON + full export
-> Wait on your screen and watch the export take place. You'll see it disappear before you can even download it.

**Comment by richard-julien** [comment]
Can you retry on latest version @Lhorus6 , i think i have fixed that in another ticket. Thanks

**Comment by Lhorus6** [comment]
Indeed, it works! So I close it
